### ISSUE TITLE
Critical Issue and apk not build with pusher-channels-flutter on Flutter 3.24.0

### ISSUE THREAD

**Issue by razuahmed75** [pusher/pusher-channels-flutter#174]
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':pusher_channels_flutter:verifyReleaseResources'.
> A failure occurred while executing com.android.build.gradle.tasks.VerifyLibraryResourcesTask$Action
   > Android resource linking failed
     ERROR:/Users/bugfinder/Downloads/bug-finder-project/PAYSECURE-APP/paysecure/build/pusher_channels_flutter/intermediates/merged_res/release/values/values.xml:194: AAPT: error: resource android:attr/lStar not found.
     
     
I'm reaching out to report an issue with the pusher-channels-flutter package while trying to build an APK on Flutter 3.24.0. Unfortunately, the build is currently failing, and it appears to be due to compatibility issues with the latest Flutter version. I have tried troubleshooting, but the problem seems to persist.

Given how widely this package is used, I would highly recommend looking into this issue and providing a fix. It would greatly help many developers, including myself, who rely on this package for production environments.

**Comment by ryansyah17** [comment]
Launching lib\main.dart on sdk gphone16k x86 64 in debug mode...

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':pusher_channels_flutter'.
> Could not create an instance of type com.android.build.api.variant.impl.LibraryVariantBuilderImpl.
   > Namespace not specified. Specify a namespace in the module's build file. See https://d.android.com/r/tools/upgrade-assistant/set-namespace for information about setting the namespace.

     If you've specified the package attribute in the source AndroidManifest.xml, you can use the AGP Upgrade Assistant to migrate to the namespace value in the build file. Refer to https://d.android.com/r/tools/upgrade-assistant/agp-upgrade-assistant for general information about using the AGP Upgrade Assistant.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.

BUILD FAILED in 1s
Error: Gradle task assembleDebug failed with exit code 1

Exited (1).

**Comment by jim-2000** [comment]
## Using pusher_channels_flutter from GitHub

To use the `pusher_channels_flutter` package in your Flutter project, you can add it directly from the GitHub repository.

### Option 1: Regular Dependency
Add the following to your `pubspec.yaml` under `dependencies`:
```yaml
dependencies:
  flutter:
    sdk: flutter
  pusher_channels_flutter:
    git:
      url: https://github.com/ArmanKT/pusher-channels-flutter.git
      ref: master

or 
dependency_overrides:
  pusher_channels_flutter:
    git:
      url: https://github.com/ArmanKT/pusher-channels-flutter.git
      ref: master

**Comment by ysushah** [comment]
FAILURE: Build failed with an exception.
[   +2 ms] * What went wrong:
[        ] Execution failed for task ':pusher_channels_flutter:verifyReleaseResources'.
[        ] > A failure occurred while executing com.android.build.gradle.tasks.VerifyLibraryResourcesTask$Action
[        ]    > Android resource linking failed
[   +1 ms]      ERROR:D:\SAMIRBHAI\e-school-saas-main\build\pusher_channels_flutter\intermediates\merged_res\release\values\values.xml:194: AAPT: error: resource android:attr/lStar not found.
[   +1 ms] * Try:
[        ] > Run with --debug option to get more log output.
[   +1 ms] > Run with --scan to get full insights.
[        ] * Exception is:
[        ] org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':pusher_channels_flutter:verifyReleaseResources'.
[        ]      at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.lambda$executeIfValid$1(ExecuteActionsTaskExecuter.java:147)
[   +1 ms]      at org.gradle.internal.Try$Failure.ifSuccessfulOrElse(Try.java:282)
[        ]      at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeIfValid(ExecuteActionsTaskExecuter.java:145)
[        ]      at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:133)
[        ]      at org.gradle.api.internal.tasks.execution.CleanupStaleOutputsExecuter.execute(CleanupStaleOutputsExecuter.java:77)
[        ]      at org.gradle.api.internal.tasks.execution.FinalizePropertiesTaskExecuter.execute(FinalizePropertiesTaskExecuter.java:46)
[        ]      at org.gradle.api.internal.tasks.execution.ResolveTaskExecutionModeExecuter.execute(ResolveTaskExecutionModeExecuter.java:51)
[        ]      at org.gradle.api.internal.tasks.execution.SkipTaskWithNoActionsExecuter.execute(SkipTaskWithNoActionsExecuter.java:57)
[        ]      at org.gradle.api.internal.tasks.execution.SkipOnlyIfTaskExecuter.execute(SkipOnlyIfTaskExecuter.java:56)
[   +1 ms]      at org.gradle.api.internal.tasks.execution.CatchExceptionTaskExecuter.execute(CatchExceptionTaskExecuter.java:36)
[        ]      at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.executeTask(EventFiringTaskExecuter.java:77)
[        ]      at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.call(EventFiringTaskExecuter.java:55)
[        ]      at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.call(EventFiringTaskExecuter.java:52)
[        ]      at org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:204)
[  +10 ms]      at org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:199)
[   +1 ms]      at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:66)
[        ]      at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:59)
[        ]      at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:157)
[        ]      at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:59)
[        ]      at org.gradle.internal.operations.DefaultBuildOperationRunner.call(DefaultBuildOperationRunner.java:53)
[        ]      at org.gradle.internal.operations.DefaultBuildOperationExecutor.call(DefaultBuildOperationExecutor.java:73)
[        ]      at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter.execute(EventFiringTaskExecuter.java:52)
[   +1 ms]      at org.gradle.execution.plan.LocalTaskNodeExecutor.execute(LocalTaskNodeExecutor.java:74)
[        ]      at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$InvokeNodeExecutorsAction.execute(DefaultTaskExecutionGraph.java:333)
[        ]      at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$InvokeNodeExecutorsAction.execute(DefaultTaskExecutionGraph.java:320)
[        ]      at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$BuildOperationAwareExecutionAction.execute(DefaultTaskExecutionGraph.java:313)
[        ]      at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$BuildOperationAwareExecutionAction.execute(DefaultTaskExecutionGraph.java:299)
[   +5 ms]      at org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker.lambda$run$0(DefaultPlanExecutor.java:143)
[   +2 ms]      at org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker.execute(DefaultPlanExecutor.java:227)
[  +11 ms]      at org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker.executeNextNode(DefaultPlanExecutor.java:218)
[  +14 ms]      at org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker.run(DefaultPlanExecutor.java:140)


I'm reaching out to report an issue with the pusher-channels-flutter package while trying to build an APK on Flutter 3.24.0. Unfortunately, the build is currently failing, and it appears to be due to compatibility issues with the latest Flutter version. I have tried troubleshooting, but the problem seems to persist.

Given how widely this package is used, I would highly recommend looking into this issue and providing a fix. It would greatly help many developers, including myself, who rely on this package for production environments.

**Comment by aonemd** [comment]
Hello, we just published a new release `2.3.0` to upgrade the dependencies.
